agentic/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@editor-extensions/agentic",
+  "version": "0.1.0",
+  "private": true,
+  "type": "module",
+  "types": "./dist/index.d.ts",
+  "main": "./dist/index.js",
+  "exports": {
+    ".": {
+      "types": "./dist/index.d.ts",
+      "import": "./dist/index.js",
+      "default": "./dist/index.js"
+    },
+    "./package.json": "./package.json"
+  },
+  "scripts": {
+    "clean": "rimraf dist",
+    "ci": "npx jest",
+    "lint": "eslint .",
+    "lint:fix": "eslint --fix .",
+    "prebuild": "npm run clean",
+    "build": "npm run prebuild && tsup src/index.ts --dts --format esm --out-dir dist --clean",
```

Comment:
npm runs the prebuild script before the build script, so explicitly calling it should be redundant